packages/snaps-simulator/src/features/polling/sagas.ts

```diff
@@ -1,4 +1,4 @@
-import { detectSnapLocation } from '@metamask/snaps-controllers/dist/snaps/location';
+import { detectSnapLocation } from '@metamask/snaps-controllers';
```

Comment:
Does this not balloon the bundle size?

Reply:
Doesn't seem to make any difference. If we don't change it, building the simulator depends on build output (as `/dist` doesn't exist otherwise).